pkg/kubernetes/scripts/registry.js

```diff
@@ -163,6 +164,59 @@
                 return select().kind("Project");
             };
 
+            // TODO: invalidate cache on role changes
+            var sarCache = { };
+
+            $scope.subjectAccessReview = function subjectAccessReview(project, user, verb, resource) {
```

Comment:
This will be needed from other pages too, should be in policy.js so it can be used elsewhere.